content/src/controller.js

```diff
@@ -1769,6 +1773,22 @@ function cancelAndClearPosition(pos) {
   modelatpos(pos).running = false;
 }
 
+function instrumentCode(code, mimetype) {
+  if (/javascript/.test(mimetype)) {
```

Comment:
/^javascript$/i.test

Mime types are case insensitive (so use i). Regular expression matching will match a substring unless you anchor the ends with ^ and $.